library/src/armory/model/object_detection/object_detector.py

```diff
@@ -36,7 +38,7 @@ class ObjectDetector(ArmoryModel, ModelProtocol):
     def __init__(
         self,
         name: str,
-        model,
+        model: nn.Module,
```

Comment:
I don't believe anything in this class assumes that the model is or has to be a PyTorch model, which is why the type hint was previously omitted.

Reply:
I'll remove the `model` typing hint.